docs/components/card/README.md

```diff
@@ -120,10 +111,10 @@ CdrCards should always be used to link to other content, and the `cdr-card__link
 
 ## Accessibility
 
-To ensure that usage of this component complies with accessibility guidelines, do the following:
+Many WCAG requierments are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responcible for the following:
```

Comment:
"responsible"

what happened to the article/section/div info? is that not applicable?

Reply:
the component does not support it yet - so I backlogged it :P -man my spelling game is taking a nosedive

Reply:
what do you mean by "support it"? wouldn't the consumer just wrap the card content in that tag?

Reply:
well the card as it is auto uses the article tag - I think we will want to update this to something like text where the tag defaults to article but can be set to section or div